src/Main.elm

```diff
@@ -307,6 +311,7 @@ view model =
                 Nothing ->
                     []
     in
+    if model.isDifficultySet == True then
```

Comment:
`model.isDifficultySet == True` evaluates to type `Bool`, but `model.isDifficultySet` is already a `Bool`!

instead of

```elm
if model.isDifficultySet == True then
```

you can just do

```elm
if model.isDifficultySet then
```